### ISSUE TITLE
Extracting with include_images option causes subsequent extractions to fail

### ISSUE THREAD

**Issue by TommiNieminen** [adbar/trafilatura#51]
There's a bug with extraction using include_images option, it only works once and then extractions start to fail. The bug is in htmlprocessing.py, lines 46 to 50:

  ```    
    if include_images is True:
        # Many websites have <img> inside <figure> or <picture> or <source> tag
        for element in ['figure', 'picture', 'source']:
            MANUALLY_CLEANED.remove(element)
        MANUALLY_STRIPPED.remove('img')
```
This bit of code will be repeated for all extractions, but since the tags have been removed on the first extraction, a ValueError will be thrown for all other extractions. I tested that adding existence checks solves the issue:

```
if include_images is True:
        # Many websites have <img> inside <figure> or <picture> or <source> tag
        for element in ['figure', 'picture', 'source']:
            if element in MANUALLY_CLEANED:
                MANUALLY_CLEANED.remove(element)
        if 'img' in MANUALLY_STRIPPED:
            MANUALLY_STRIPPED.remove('img')
```

**Comment by adbar** [comment]
Thanks, the way the lists were used was inconsistent.
It should be fixed in 8a5123ff829c5b1d8dbd5232259942b0b596cd4b, can you confirm by running the last version from the repository?

**Comment by TommiNieminen** [comment]
Thanks, the cleaning_list removal works now, but the stripping_list removal will still cause a ValueError with repeated extractions, since the code tries to remove 'img' that was already removed during the first extraction.

```
        cleaning_list = [e for e in cleaning_list if e
                         not in ('figure', 'picture', 'source')]
        stripping_list.remove('img')
```

**Comment by adbar** [comment]
I got it wrong: without `copy()` the lists were linked to the same object and thus identical. It should be fixed now (33cd96bebb855b9e65db3d30e244cf0877495ca5).
Can you please confirm?

**Comment by TommiNieminen** [comment]
Seems to work now, thanks.
